libs/react/src/provider/FrontMcpProvider.tsx

```
@@ -0,0 +1,122 @@
+  useEffect(() => {
+    mountedRef.current = true;
+
+    if (autoConnect) {
+      connectClient();
+    }
+
+    return () => {
+      mountedRef.current = false;
+      clientRef.current = null;
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)